Pulsarc/Utils/PulsarcDiscord.cs

```diff
@@ -9,26 +8,29 @@ static class PulsarcDiscord
     {
         private const string LOGO = "pulsarc_logo";
 
-        private static double lastUpdateTime = -15000;
+        private const double LastUpdateTime = -15000;
 
-        private const double UPDATE_WAIT_TIME = 15000;
+        private const double UpdateWaitTime = 15000;
 
-        private static bool Waiting => PulsarcTime.CurrentElapsedTime < lastUpdateTime + UPDATE_WAIT_TIME && waitingTriggered;
+        private static bool Waiting => PulsarcTime.CurrentElapsedTime < LastUpdateTime + UpdateWaitTime && waitingTriggered;
         private static bool waitingTriggered = false;
 
         public static RichPresence Presence { get; private set; }
 
-        static public void Initialize()
+        public static void Initialize()
         {
-            // 604680029439393812 <- Adri's AppID, in case we go back to that one.
-            // 649491503391047701 <- FRUP's AppID, currently being used.
-            DiscordManager.CreateClient("649491503391047701", Wobble.Discord.RPC.Logging.LogLevel.None);
-            Presence = new RichPresence();
-            Presence.Assets = new Assets();
-            Presence.Timestamps = new Timestamps();
+            Dictionary<string, string> appIdSwitch = new Dictionary<string, string>()
```

Comment:
So, I'm not entirely sure what this is for. I understand being able to switch through app IDs easily, but we shouldn't be switching IDs around often anyways. The whole reason I switched from Adri's app to mine was so I could upload specific images to use with Discord; I didn't have access to his app. I just kept his down in case something was needed.

We're probably going to stick with my ID unless something changes. For most cases you don't need to make a new app to test something new with the Discord functionality. If you do, you let me know and I'll make the changes you need to the app.

Also you swapped my ID with what I presume is your ID lol.